Jenkinsfile

```diff
@@ -1,61 +1,67 @@
-def date_str_bld = new Date().format("yyyy-mm-dd")
+@Library('xmos_jenkins_shared_library@v0.18.0') _
```

Comment:
Let's remove the Jenkins and requirements.txt changes from this PR. I suspect the Python version was an issue with some particular Jenkins machine since it was working for the current xtlm_interpreter. We can add the changes back in if there are issues.

Reply:
In this case, this Jenkinsfile rewrite was done to use Jenkins in line with the way the software team uses Jenkins. This file now uses the Xmos_jenkins_shared_library, and uses a virtual environment with help from this library.
So this change may not be necessary, but I think it is better practice to do it this way. Let me know your thoughts on this too!

Reply:
Alright, that's fine in that case. Thanks!